packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -162,6 +163,8 @@ export default class MeetingRequest extends StatelessWebexPlugin {
         url: deviceUrl,
         // @ts-ignore - config comes from registerPlugin
         deviceType: this.config.meetings.deviceType,
+        // TODO
```

Comment:
nitpick: I'd prefer there to be an explanation of why there is a todo here if it really does need to stay - I'm not convinced it needs to be in the source right now, but it is not obvious as to why it is without looking at this PR.

Reply:
Sure - i can link to the jira or I can just remove the TODO, that is fine.